lib/lives.rb

```diff
@@ -0,0 +1,11 @@
+class Lives
+  attr_reader :lives
+
+  def initialize
+    @lives = 8
```

Comment:
I'd pass this in with a default value here, so it will be the number specified, or fall back to 8. 

You could do this with

```
def initialize(lives=8)
	@lives = lives
end
```

Reply:
Thanks Dan we will add this in now.